json-guides/bulkhead.json

```diff
@@ -143,12 +146,12 @@
         {   "name": "playground",
             "title": "Interactive bulkhead playground",
             "description": [
-              "Now that you learned about bulkhead, you can explore the parameters in the <code>@Bulkhead</code> annotation and see the bulkhead in action.",
-              "<br>You learned about the following parameters:",
-              "<ul>",
-              "<li><b>value</b>: ",
-              "<li><b>waitingTaskQueue</b>: ",
-              "</ul>"
+                "Now that you learned about bulkhead, you can explore the parameters in the <code>@Bulkhead</code> annotation and see the bulkhead in action.",
```

Comment:
How about the @Asynchronous annotation?

Reply:
Fixed. `@Asynchronous` itself doesn't have any parameters, but I've mentioned it in the updated sentence.